kafkajobs/opts.go

```diff
@@ -65,6 +65,7 @@ type config struct {
 	// global
 	Brokers []string `mapstructure:"brokers"`
 	SASL    *SASL    `mapstructure:"sasl"`
+	Ping    *Ping    `mapstructure:"ping"`
```

Comment:
Let's use `ping_timeout` and PingTimeout field (`time.Duration`) type. No need to have additional structure for the 1 field 😃

Reply:
I assumed that in the future ping by interval may appear and this could lie in Ping.Interval

Reply:
Hm, you're right, it would be nice to have some kind of healtchecks. Ok, let it be a separate structure 👍🏻